AndelaOSP/art-dashboard
# [#157340066] Load all asset make dropdown asset types

## What does this PR do?
Temporarily fixes the dropdown pagination issue for asset types on asset make modal component.

## Description of Task to be completed?
Enable retrieving of all asset types to be loaded on the dropdown by bypassing the pagination.

## How should this be manually tested?
On the assets page, click on the + icon for asset makes and on the displayed modal click on the asset types drop down. This should show all existing asset types.

## What are the relevant pivotal tracker stories?
157340066

## Any background context you want to add?
N/A
## Important notes
N/A
## Packages installed
N/A
## Deployment note
N/A
## Related PRs branch | PR (branch_name) | (pr_link)
Branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()
## Todos
## Screenshots
<img width="473" alt="screen shot 2018-07-10 at 15 39 19" src="https://user-images.githubusercontent.com/27014080/42510404-7331b210-8457-11e8-9e45-979b380802b4.png">
